agent/api/ecsclient/client.go

```diff
@@ -191,11 +201,40 @@ func (client *APIECSClient) registerContainerInstance(clusterRef string, contain
 
 	resp, err := client.standardClient.RegisterContainerInstance(&registerRequest)
 	if err != nil {
-		log.Error("Could not register", "err", err)
+		log.Error("Could not register", "error", err)
```

Comment:
logger is used throughout this file. There are 12 calls into the `log` object in master, and one to seelog. I think the right thing would be to convert this whole file to seelog, but it's outside the scope for this change. I don't want to add to the inconsistency, so I'd like to stick with `logger` for now.